go/api/response_handlers.go

```diff
@@ -31,6 +31,76 @@ func handleStringOrNullResponse(response *C.struct_CommandResponse) string {
 	return handleStringResponse(response)
 }
 
+// handleIntegerOrNullResponse handles the response from a command that returns an integer or 0.
+func handleIntegerOrNullResponse(response *C.struct_CommandResponse) int64 {
+	if response == nil {
+		return 0
+	}
+
+	return int64(response.int_value)
+}
+
+// handleBooleanResponse converts a C struct_CommandResponse's bool_value to a Go bool.
+func handleBooleanResponse(response *C.struct_CommandResponse) bool {
```

Comment:
we should pass to the caller that an error occurred. If `null` is acceptable, we should change this method to:

```go
handleBooleanOrNullResponse
```

Reply:
Why are we diverting from go's function convention? I'd rather have all these `handleYYYResponse` methods signagure changed into:

```go
func handleSomethingResponse(response *C.struct_CommandResponse) (Something, error) {
...
}
```

It will make our coding much easier